usr/share/bash-completion/completions/kapacitor

```diff
@@ -49,7 +48,7 @@ _kapacitor()
                         words=$(_kapacitor_dbrps)
                         ;;
                     -tick)
-                        COMPREPLY=($(compgen -o filenames -A file -X '!*.tick' -- "$cur"))
+			_kapacitor_tick_files "${cur}"
```

Comment:
The -tick is also part of the `define-template` section. Can you add this fix there too?